commands/mentor.py

```diff
@@ -0,0 +1,28 @@
+import requests
+import random
+from disco.bot import Bot,Plugin
+from disco.types.message import MessageEmbed
+from config_mock import AnnounceBotConfig, TestPluginConfig
+
+MENTOR_ID = 487807730506006540
+
+@Plugin.with_config(TestPluginConfig)
+class MentorPlugin(Plugin):
+
+    def get_avail_mentors(self):
+       return [u.user.id for u in self.bot.client.state.guilds.get(487611837664198679).members.values() if MENTOR_ID in u.roles and u.user.presence.status != "OFFLINE"]
+
+    def ping_mentor(self, mentor, author, content):
+        self.bot.client.state.channels.get(self.config.channel_IDs.get('MENTOR_CHANNEL')).send_message(self.config.messages.get("help_message").format(mentor, author, content))
+
+    def concat_name_discrim(self, name, discrim):
+        return "{}#{}".format(name, discrim)
```

Comment:
works but technically not needed, if you do ``str(user)`` the lib does this for you

Reply:
omg, sick! thanks!